oauth2client/contrib/gce.py

```diff
@@ -123,3 +127,60 @@ def create_scoped_required(self):
 
     def create_scoped(self, scopes):
         return AppAssertionCredentials(scopes, **self.kwargs)
+
+    def sign_blob(self, blob):
+        """Cryptographically sign a blob (of bytes).
+
+        This method is provided to support a common interface, but
```

Comment:
Well `creds.sign_blob(blob)` is the common interface, it just never crosses the finish line by allowing

```python
key_id, signed_bytes = creds.sign_blob(blob)
```